src/tests/ftest/container/rebuild_tests.yaml

```diff
@@ -0,0 +1,29 @@
+hosts:
+ test_machines:
+  - boro-A
+  - boro-B
+  - boro-C
+  - boro-D
+  - boro-E
+  - boro-F
+timeout: 700
+server:
+ server_group: daos_server
+testparams:
+ ranks: !mux
+  rank1:
+   rank: 5
+  rank4:
+   rank: 4
+  rank3:
+   rank: 3
+ createmode:
```

Comment:
use standard pool parameter pattern, and just generally, remember this is a tree like a directory structure and you create different branches for different parameter groups.

Reply:
Changed to match the example I was pinged in Skype:

`poolparams:
   createmode:
     mode: 511
   createset:
     setname: daos_server
   createsize:
     size: 1073741824`